src/main/kotlin/com/zepben/evolve/services/network/NetworkServiceComparator.kt

```diff
@@ -370,6 +370,9 @@ class NetworkServiceComparator @JvmOverloads constructor(
     private fun ObjectDifference<out ConnectivityNodeContainer>.compareConnectivityNodeContainer(): ObjectDifference<out ConnectivityNodeContainer> =
         apply { comparePowerSystemResource() }
 
+    private fun ObjectDifference<out Curve>.compareCurve(): ObjectDifference<out Curve> =
+        apply { compareIdentifiedObject() }
```

Comment:
todo: should be checking curve data. You need to check why this is not being detected in the tests.

Reply:
Test wasn't added in the comparator test before. Has been added and is also now checking curvedata.